2020/Day8.py

```diff
@@ -0,0 +1,120 @@
+# https://adventofcode.com/2020/day/8
+import copy
+
+DAY = 8
+
+x_test = '''nop +0
+acc +1
+jmp +4
+acc +3
+jmp -3
+acc -99
+acc +1
+jmp -4
+acc +6'''.split('\n')
+y_test = 5
+
+y_test2 = 8
+
+
+# Part 1
+def parse_lines(INPUT):
+    output = []
+    for i in INPUT:
+        i = i.split()
+        output.append([i[0], int(i[1])])
+
+    return output
+
+
+def execute_program(code):
+    restart = False
+    visited = []
```

Comment:
I think it's slightly more efficient to use a set here. Containment in a set is faster than a list I think but likely not a huge difference. I'm just all about them sets

Reply:
I know you love sets, but I wanted to preserve the order they were visited.

Reply:
That's fair, my solution doesn't actually provide the path you took.